### PR TITLE
Use Istio gateway instead of Kubernetes ingress

### PR DESCRIPTION
**Pull request by gytis**
https://issues.jboss.org/browse/SB-756

The only issue I see with this approach, is that gateway is configured with a hardcoded service names, which might be different when creating application via launcher.

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -62,15 +57,21 @@ oc new-app --template=spring-boot-istio-security-name -p SOURCE_REPOSITORY_URL=h
 oc new-app --template=spring-boot-istio-security-greeting -p SOURCE_REPOSITORY_URL=https://github.com/snowdrop/spring-boot-istio-security-booster -p SOURCE_REPOSITORY_REF=master -p SOURCE_REPOSITORY_DIR=spring-boot-istio-security-greeting
 ```
 
+== Configure Istio gateway
+This application uses v1alpha3 routing API. Execute the following command to configure gateway and virtual service:
+```bash
+oc apply -f rules/gateway.yml
```
Comment (geoand) [inline README.adoc]: When I tried this in the Distributed Tracing booster, I needed to specify the namespace as well like:

`oc apply -f rules/gateway.yml -n $(oc project -q)`

If I didn't specify the namespace, the gateway wasn't working correctly

Comment (gytis) [reply README.adoc]: Worked fine for me without `-n`

Comment (gytis) [reply README.adoc]: It gues applies it to the current project.

Comment (geoand) [reply README.adoc]: I will need re-check. Perhaps the fact that I was using `oc create` instead of `oc apply` had something to do with it

**Comment by gytis** [comment]
@geoand can we merge this?

**Pull request merged**